src/plugins/poster/poster.js

```diff
@@ -98,7 +98,9 @@ export default class PosterPlugin extends UIContainerPlugin {
       this.container.disableMediaControl()
       this.$el.show()
     } else {
-      this.container.enableMediaControl()
+      if (this.container.playback.name !== 'no_op') {
+        this.container.enableMediaControl()
+      }
```

Comment:
If #1169 is merged then these 3 lines won't be needed as a special case, as even if the media control is enabled it won't appear.
I think it's better if the poster plugin isn't responsible for hiding the media control because some users might not use the poster plugin and the media control should still not be present in that case for the playback type `NO_OP`.